pkg/odo/cli/service/list_operator.go

```diff
@@ -27,10 +27,10 @@ type clusterInfo struct {
 type serviceItem struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-	ClusterInfo       *clusterInfo              `json:"clusterInfo,omitempty"`
-	InDevfile         bool                      `json:"inDevfile"`
-	Deployed          bool                      `json:"deployed"`
-	Manifest          unstructured.Unstructured `json:"manifest"`
+	ClusterInfo       *clusterInfo           `json:"clusterInfo,omitempty"`
+	InDevfile         bool                   `json:"inDevfile"`
+	Deployed          bool                   `json:"deployed"`
+	Manifest          map[string]interface{} `json:"manifest"`
```

Comment:
To avoid a conversion from map[string]interface{} to Unstructured, at this point if I remember well:
```
if log.IsJSON() {
			svcFullName := strings.Join([]string{b.CustomResource, o.ServiceName}, "/")
			svc := NewServiceItem(svcFullName)
			svc.Manifest = b.CustomResourceDefinition
			svc.InDevfile = true
			machineoutput.OutputSuccess(svc)
		}
```